src/main/telemetry/srxl.c

```diff
@@ -194,15 +194,11 @@ uint16_t getMotorAveragePeriod(void)
 
 #if defined(USE_DSHOT_TELEMETRY)
     if (useDshotTelemetry) {
-        uint16_t motors = getMotorCount();
-
-        if (motors > 0) {
-            for (int motor = 0; motor < motors; motor++) {
-                rpm += getDshotTelemetry(motor);
-            }
-            rpm = 100.0f / (motorConfig()->motorPoleCount / 2.0f) * rpm;  // convert erpm freq to RPM.
-            rpm /= motors;           // Average combined rpm
+        rpm = 0;
```

Comment:
Good catch. Does this make `DSHOT_TELEMETRY` rpm value override `ESC_SENSOR_TELEMETRY`?

Reply:
Fixed